cpp/src/link_analysis/pagerank.cu

```diff
@@ -217,15 +217,16 @@ gdf_error gdf_pagerank_impl (gdf_graph *graph,
   GDF_REQUIRE( pagerank->data != nullptr , GDF_INVALID_API_CALL );
   GDF_REQUIRE( pagerank->null_count == 0 , GDF_VALIDITY_UNSUPPORTED );
   GDF_REQUIRE( pagerank->size > 0 , GDF_INVALID_API_CALL );
-
-  int m=pagerank->size, nnz = graph->edgeList->src_indices->size, status = 0;
+  */
+  //int m=pagerank->size, nnz = graph->edgeList->src_indices->size, status = 0;
+  int m=pagerank->size, nnz = graph->transposedAdjList->indices->size, status = 0;
   WT *d_pr, *d_val = nullptr, *d_leaf_vector = nullptr;
   WT res = 1.0;
   WT *residual = &res;
 
-  if (graph->transposedAdjList == nullptr) {
+  /*if (graph->transposedAdjList == nullptr) {

```

Comment:
Tansposed adjacency list already added in python wrapper before calling cpp pagerank

Reply:
agreed